validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -246,7 +301,8 @@ def __init__(self,
                     identity=identity,
                     secured=secured,
                     server_public_key=server_public_key,
-                    server_private_key=server_private_key)
+                    server_private_key=server_private_key,
+                    heartbeat=heartbeat)
```

Comment:
It appears to me that the dealer socket can never send a PingRequest, but Connections are always Dealers so the _send_heartbeat is getting setup on the event loop but is a conditional that is always false and then an asyncio.sleep.